ee/tasks/replay_summaries.py

```diff
@@ -0,0 +1,45 @@
+from typing import Any
+
+import structlog
+from celery import shared_task, chord
+
+from ee.session_recordings.ai.generate_embeddings import (
+    fetch_recordings_without_embeddings,
+    generate_recording_embeddings,
+)
+from posthog import settings
+from posthog.tasks.utils import CeleryQueue
+
+logger = structlog.get_logger(__name__)
+
+
+@shared_task(ignore_result=False, queue=CeleryQueue.SESSION_REPLAY_EMBEDDINGS.value)
+def embed_single_recording(session_id: str, team_id: int) -> None:
+    generate_recording_embeddings(session_id, team_id)
+
+
+@shared_task(ignore_result=True)
+def generate_recordings_embeddings_batch_on_complete(args: Any, kwargs: Any) -> None:
+    logger.info("Embeddings generation batch completed", args=args, kwargs=kwargs)
```

Comment:
I'm guessing this is where we can call `flush_embeddings_to_clickhouse` given it is passed the results of the individual `embed_single_recording` tasks

Reply:
I really like this approach, makes the generation very simple (at least for one team)